terminus-ui/button/src/button.component.ts

```diff
@@ -325,16 +325,16 @@ export class TsButtonComponent implements OnInit, OnDestroy {
 
 
   /**
-   * Do something when clicked
+   * Do something when clickedButton
```

Comment:
Let's reword this to something more english friendly.. maybe `Handle button clicks` or something